posthog/clickhouse/test/__snapshots__/test_schema.ambr

```diff
@@ -228,7 +228,7 @@
       distinct_id VARCHAR,
       person_id UUID,
       is_deleted Int8,
-      version Int64 DEFAULT 1
```

Comment:
version beginning with `0` is ok as long as this is used as `version` and not `sign` for the table declaration

Reply:
we use version 0 explicitly writing it from plugin-server, so I can't see how this would be a problem, e.g. https://metabase.posthog.net/question#eyJkYXRhc2V0X3F1ZXJ5Ijp7InR5cGUiOiJuYXRpdmUiLCJuYXRpdmUiOnsicXVlcnkiOiJzZWxlY3QgKiBmcm9tIHBlcnNvbl9kaXN0aW5jdF9pZDIgd2hlcmUgdmVyc2lvbiA9IDAgbGltaXQgNSIsInRlbXBsYXRlLXRhZ3MiOnt9fSwiZGF0YWJhc2UiOjh9LCJkaXNwbGF5IjoidGFibGUiLCJwYXJhbWV0ZXJzIjpbXSwidmlzdWFsaXphdGlvbl9zZXR0aW5ncyI6e319